### ISSUE TITLE
Any chance we could get a screenshot?

### ISSUE THREAD

**Issue by dunderpate** [Koshroy/darwaza#1]
Hey there!

Thanks for making this :D I just wondered if there's a chance you could add a screenshot to the README.md. I think it would help people get excited about the project if they could see what it looked like.

Dunderpate

**Comment by Koshroy** [comment]
Absolutely. I'm sorry for responding to this so late, I'll add a screenshot.
